xml/System.ServiceModel.Channels/ServiceChannelProxy.xml

```diff
@@ -1,1536 +1,1540 @@
+      <MemberSignature Language="C++ CLI" Value="property System::ServiceModel::EndpointAddress ^ System::ServiceModel::IContextChannel::LocalAddress { System::ServiceModel::EndpointAddress ^ get(); };" />
+      <MemberType>Property</MemberType>
+      <Implements>
+        <InterfaceMember>P:System.ServiceModel.IContextChannel.LocalAddress</InterfaceMember>
+      </Implements>
+      <AssemblyInfo>
+        <AssemblyName>System.ServiceModel.Primitives</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.ServiceModel.EndpointAddress</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Gets the local endpoint for the channel.</summary>
+        <value>The local <see cref="T:System.ServiceModel.EndpointAddress" /> for the channel.</value>
+        <remarks>To be added.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="System.ServiceModel.IContextChannel.OperationTimeout">
+      <MemberSignature Language="C#" Value="TimeSpan System.ServiceModel.IContextChannel.OperationTimeout { get; set; }" />
+      <MemberSignature Language="ILAsm" Value=".property instance valuetype System.TimeSpan System.ServiceModel.IContextChannel.OperationTimeout" />
+      <MemberSignature Language="DocId" Value="P:System.ServiceModel.Channels.ServiceChannelProxy.System#ServiceModel#IContextChannel#OperationTimeout" />
+      <MemberSignature Language="VB.NET" Value=" Property OperationTimeout As TimeSpan Implements IContextChannel.OperationTimeout" />
+      <MemberSignature Language="F#" Value="member this.System.ServiceModel.IContextChannel.OperationTimeout : TimeSpan with get, set" Usage="System.ServiceModel.IContextChannel.OperationTimeout" />
+      <MemberSignature Language="C++ CLI" Value="property TimeSpan System::ServiceModel::IContextChannel::OperationTimeout { TimeSpan get(); void set(TimeSpan value); };" />
+      <MemberType>Property</MemberType>
+      <Implements>
+        <InterfaceMember>P:System.ServiceModel.IContextChannel.OperationTimeout</InterfaceMember>
+      </Implements>
+      <AssemblyInfo>
+        <AssemblyName>System.ServiceModel.Primitives</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.TimeSpan</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Gets or sets the time period within which an operation must complete or an exception is thrown.</summary>
+        <value>The <see cref="T:System.TimeSpan" /> that specifies the time period within which an operation must complete.</value>
+        <remarks>This member is an explicit interface member implementation. It can be used only when the &lt;xref:System.ServiceModel.Channels.ServiceChannelProxy&gt; instance is cast to an &lt;xref:System.ServiceModel.IContextChannel&gt; interface.</remarks>

```

Comment:
```suggestion
        <remarks>This member is an explicit interface member implementation. It can be used only when the <see cref="T:System.ServiceModel.Channels.ServiceChannelProxy"/> instance is cast to an <see cref="T:System.ServiceModel.IContextChannel"/> interface.</remarks>

```